resync/tools/check_public_dependencies.py

```
@@ -3,7 +3,7 @@
 import ast
 import os
 from dataclasses import dataclass
-from typing import Iterable, List, Tuple
+from typing import Iterable, List
```

Ruff (0.15.0)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)